strings/strings.json

```diff
@@ -7438,6 +7438,14 @@
     "en": "Sort results by {0}",
     "fr": "Trier les résultats par {0}"
   },
+  "SortResultsByAscending": {
+    "en": "Sort results by {0} in ascending order",
```

Comment:
Could we remove the `results` from the string ? 

Makes it a bit more generic for other use case (eg: product search).

"Sort by {0} in ascending order" seems just as valid.

Reply:
Sure, that sounds better to me.